Plan/common/src/main/java/com/djrapitops/plan/storage/database/sql/tables/UserInfoTable.java

```diff
@@ -42,14 +42,16 @@
     public static final String REGISTERED = "registered";
     public static final String OP = "opped";
     public static final String BANNED = "banned";
+    public static final String HOSTNAME = "hostname";
 
     public static final String INSERT_STATEMENT = "INSERT INTO " + TABLE_NAME + " (" +
             USER_UUID + ',' +
             REGISTERED + ',' +
             SERVER_UUID + ',' +
             BANNED + ',' +
+            HOSTNAME + ',' +
             OP +
-            ") VALUES (?, ?, ?, ?, ?)";
+            ") VALUES (?, ?, ?, ?, ?, ?)";
```

Comment:
The use of this statement is not updated in `LargeStoreQueries.java` which is breaking the backup and restore tests
See <https://github.com/plan-player-analytics/Plan/blob/master/Plan/common/src/main/java/com/djrapitops/plan/storage/database/queries/LargeStoreQueries.java#L220>